packages/components/stories/List.js

```diff
@@ -428,19 +397,59 @@ storiesOf('List', module)
 		return (
 			<div style={{ height: '60vh' }} className="virtualized-list">
 				<h1>List</h1>
-				<p>Display a list with columns containing actions.</p>
+				<p>A column can contains only actions that appear on mouseover.</p>
 				<IconsProvider defaultIcons={icons} />
 				<List {...columnActionsProps} virtualized />
 			</div>
 		);
 	})
+	.add('Virtualized - selection', () => {
+		const selectedItemsProps = Immutable.fromJS(props).toJS();
```

Comment:
Why not use something like lodash's cloneDeep. It avoid's moving from a mutable to an immutable data structure and back.